discovery-provider/plugins/notifications/src/processNotifications/mappers/usdcPurchaseSeller.ts

```diff
@@ -100,7 +100,7 @@ export class USDCPurchaseSeller extends BaseNotification<USDCPurchaseSellerRow>
     const buyerHandle = users[this.buyerUserId]?.handle
     const sellerUsername = users[this.notificationReceiverUserId]?.name
     const sellerHandle = users[this.notificationReceiverUserId]?.handle
-    const price = this.amount
+    const price = this.totalAmount
```

Comment:
I must have missed where we are setting this field. Was it added before?

Reply:
I added it with the transactional email PR! (since those had needed pay extra line items)